ads/caprofitx.js

```diff
@@ -14,14 +14,14 @@
  * limitations under the License.
  */
 
-import {loadScript, checkData, validateDataExists} from '../3p/3p';
+import {loadScript, validateData} from '../3p/3p';
+
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
 export function caprofitx(global, data) {
-  checkData(data, ['tagid']);
-  validateDataExists(data, ['tagid']);
+  validateData(data, ['tagid'], []);
```

Comment:
Ah! I see this is for [] is added for `checkData()`. This is a bit confusing to me